example-jobs/SampleBuildFlow-example.groovy

```diff
@@ -1,15 +1,31 @@
 import jenkins.automation.builders.FlowJobBuilder
 
-def oahMaster= new FlowJobBuilder(
+def exampleFlow = new FlowJobBuilder(
         name: 'GeneratedFlowJob',
         description: 'this our first stab at it',
         jobs:['job1', 'job2']
 ).build(this);
 
-oahMaster.with{
+exampleFlow.with{
   logRotator{
       numToKeep(365)
   }
+}
+
+def exampleFlowWithWorkspace = new FlowJobBuilder(
+        name: 'flow-job-with-workspace',
+        description: 'a build flow job with its own workspace',
+        jobs:['job1', 'job2']
+).build(this)
+
+exampleFlowWithWorkspace.with{
+    scm {
+        git('https://github.com/cfpb/qu.git', 'master')
+    }
+
+    configure { node ->
+        node << buildNeedsWorkspace(true)
```

Comment:
Because flow jobs that pull from a repo need to have a workspace. Without configuring it to have a workspace, you get nasty errors in jenkins when the job runs. I've been thinking that we might even want to just set this by default... there's no harm to it when you don't need it, and when you do need it, unless you know exactly what you're missing, the error is not clear